layouts/partials/components/last-updated.html

```diff
@@ -1,9 +1,27 @@
 {{- $lastUpdated := (T "lastUpdated") | default "Last updated on" -}}
+{{- $editThisPage := (T "editThisPage") | default "Edit this page"}}
 
 {{- if site.Params.displayUpdatedDate -}}
+  <div class="mt-12 mb-8 block text-xs text-gray-500 ltr:text-right rtl:text-left dark:text-gray-400">
+  {{- if and (site.Params.editURL.enable) (eq site.Params.editURL.where "single") -}}
```

Comment:
I think we should extract this edit url piece into a partial component, since it's been used both here and in TOC template.